Fix es.po

IngrownMink4 opened the pull request:
According to @ijiki16, [my merge request has affected the compilation of the application due to some errors in the existing strings](https://github.com/maoschanz/dynamic-wallpaper-editor/pull/94#issuecomment-1333555652). I have released this patch and I hope it will fix the problem as soon as possible. And I apologize for the inconvenience caused.
 
### What's fixed:
- Fill in empty text strings
- Fix spelling mistakes
- Fixed formatting
- Fix compatibility issues
- Some strings did not correspond to the original `.pot` template. I guess it was a mistake of the previous contributor.

**Before including the patch, please check that there are no conflicts this time. I had to rewrite 60% of the text strings.**

IngrownMink4 committed:
Fix es.po 

Fill in empty text strings, fix spelling mistakes, fixed formatting and much more. [Pending approval]

ijiki16 commented:
There are still some errors
```
[1/2] Generating com.github.maoschanz.DynamicWallpaperEditor.appdata.xml_data_merge with a custom command
FAILED: data/com.github.maoschanz.DynamicWallpaperEditor.appdata.xml 
/usr/bin/meson --internal msgfmthelper ../data/com.github.maoschanz.DynamicWallpaperEditor.appdata.xml.in data/com.github.maoschanz.DynamicWallpaperEditor.appdata.xml xml ../data/../po
../data/../po/es.po:34: end-of-line within string
../data/../po/es.po:33: missing 'msgstr' section
../data/../po/es.po:34:11: syntax error
../data/../po/es.po:34: keyword "for" unknown
../data/../po/es.po:35: end-of-line within string
../data/../po/es.po:36: end-of-line within string
../data/../po/es.po:37: end-of-line within string
../data/../po/es.po:40: end-of-line within string
../data/../po/es.po:39: missing 'msgstr' section
../data/../po/es.po:40:9: syntax error
../data/../po/es.po:40: keyword "The" unknown
../data/../po/es.po:41: keyword "exactly" unknown
../data/../po/es.po:42: end-of-line within string
../data/../po/es.po:43: end-of-line within string
../data/../po/es.po:44: keyword "duraci" unknown
../data/../po/es.po:45: end-of-line within string
../data/../po/es.po:53: end-of-line within string
../data/../po/es.po:53:8: syntax error
../data/../po/es.po:54: end-of-line within string
../data/../po/es.po:57: end-of-line within string

```

Try to use gnome build App and you will see same ones ( I don't know correct way for fixing those `.op` files).

IngrownMink4 committed:
Fix es.po (again)

This patch solves this: https://github.com/maoschanz/dynamic-wallpaper-editor/pull/97#issuecomment-1336521959

IngrownMink4 commented:
> There are still some errors
> 
> ```
> [1/2] Generating com.github.maoschanz.DynamicWallpaperEditor.appdata.xml_data_merge with a custom command
> FAILED: data/com.github.maoschanz.DynamicWallpaperEditor.appdata.xml 
> /usr/bin/meson --internal msgfmthelper ../data/com.github.maoschanz.DynamicWallpaperEditor.appdata.xml.in data/com.github.maoschanz.DynamicWallpaperEditor.appdata.xml xml ../data/../po
> ../data/../po/es.po:34: end-of-line within string
> ../data/../po/es.po:33: missing 'msgstr' section
> ../data/../po/es.po:34:11: syntax error
> ../data/../po/es.po:34: keyword "for" unknown
> ../data/../po/es.po:35: end-of-line within string
> ../data/../po/es.po:36: end-of-line within string
> ../data/../po/es.po:37: end-of-line within string
> ../data/../po/es.po:40: end-of-line within string
> ../data/../po/es.po:39: missing 'msgstr' section
> ../data/../po/es.po:40:9: syntax error
> ../data/../po/es.po:40: keyword "The" unknown
> ../data/../po/es.po:41: keyword "exactly" unknown
> ../data/../po/es.po:42: end-of-line within string
> ../data/../po/es.po:43: end-of-line within string
> ../data/../po/es.po:44: keyword "duraci" unknown
> ../data/../po/es.po:45: end-of-line within string
> ../data/../po/es.po:53: end-of-line within string
> ../data/../po/es.po:53:8: syntax error
> ../data/../po/es.po:54: end-of-line within string
> ../data/../po/es.po:57: end-of-line within string
> ```
> 
> Try to use gnome build App and you will see same ones ( I don't know correct way for fixing those `.op` files).

I think it is now completely solved. Could you check it? @ijiki16

IngrownMink4 committed:
Update es.po

ijiki16 commented:
```
[2/2] Generating com.github.maoschanz.DynamicWallpaperEditor.desktop_data_merge with a custom command
FAILED: data/com.github.maoschanz.DynamicWallpaperEditor.desktop 
/usr/bin/meson --internal msgfmthelper ../data/com.github.maoschanz.DynamicWallpaperEditor.desktop.in data/com.github.maoschanz.DynamicWallpaperEditor.desktop desktop ../data/../po
../data/../po/es.po:540: duplicate message definition...
../data/../po/es.po:46: ...this is the location of the first definition
../data/../po/es.po:545: end-of-line within string
../data/../po/es.po:544: missing 'msgstr' section
../data/../po/es.po:545:5: syntax error
../data/../po/es.po:545: keyword "a" unknown
../data/../po/es.po:546: end-of-line within string
../data/../po/es.po:547: end-of-line within string
../data/../po/es.po:548: end-of-line within string
../data/../po/es.po:554:1: syntax error
../data/../po/es.po:554: duplicate message definition...
../data/../po/es.po:58: ...this is the location of the first definition
../data/../po/es.po:558: duplicate message definition...
../data/../po/es.po:62: ...this is the location of the first definition
../data/../po/es.po:563: duplicate message definition...
../data/../po/es.po:67: ...this is the location of the first definition
../data/../po/es.po:568: duplicate message definition...
../data/../po/es.po:72: ...this is the location of the first definition
../data/../po/es.po:573: duplicate message definition...
../data/../po/es.po:77: ...this is the location of the first definition
../data/../po/es.po:577: duplicate message definition...
../data/../po/es.po:81: ...this is the location of the first definition
../data/../po/es.po:581: duplicate message definition...
../data/../po/es.po:85: ...this is the location of the first definition
../data/../po/es.po:586: duplicate message definition...
../data/../po/es.po:90: ...this is the location of the first definition
../data/../po/es.po:590: duplicate message definition...
../data/../po/es.po:94: ...this is the location of the first definition
../data/../po/es.po:594: duplicate message definition...
../data/../po/es.po:98: ...this is the location of the first definition
../data/../po/es.po:598: duplicate message definition...
../data/../po/es.po:102: ...this is the location of the first definition
../data/../po/es.po:602: duplicate message definition...
../data/../po/es.po:106: ...this is the location of the first definition
../data/../po/es.po:606: duplicate message definition...
../data/../po/es.po:110: ...this is the location of the first definition
../data/../po/es.po:610: duplicate message definition...
../data/../po/es.po:114: ...this is the location of the first definition
../data/../po/es.po:614: duplicate message definition...
../data/../po/es.po:118: ...this is the location of the first definition
../data/../po/es.po:618: duplicate message definition...
../data/../po/es.po:122: ...this is the location of the first definition
../data/../po/es.po:622: duplicate message definition...
../data/../po/es.po:126: ...this is the location of the first definition
../data/../po/es.po:626: duplicate message definition...
../data/../po/es.po:130: ...this is the location of the first definition
../data/../po/es.po:631: duplicate message definition...
../data/../po/es.po:134: ...this is the location of the first definition
../data/../po/es.po:635: duplicate message definition...
../data/../po/es.po:138: ...this is the location of the first definition
../data/../po/es.po:639: duplicate message definition...
../data/../po/es.po:142: ...this is the location of the first definition
../data/../po/es.po:643: duplicate message definition...
../data/../po/es.po:146: ...this is the location of the first definition
../data/../po/es.po:647: duplicate message definition...
../data/../po/es.po:150: ...this is the location of the first definition
../data/../po/es.po:651: duplicate message definition...
../data/../po/es.po:154: ...this is the location of the first definition
../data/../po/es.po:655: duplicate message definition...
../data/../po/es.po:158: ...this is the location of the first definition
../data/../po/es.po:660: duplicate message definition...
../data/../po/es.po:162: ...this is the location of the first definition
../data/../po/es.po:664: duplicate message definition...
../data/../po/es.po:166: ...this is the location of the first definition
../data/../po/es.po:669: duplicate message definition...
../data/../po/es.po:170: ...this is the location of the first definition
../data/../po/es.po:674: duplicate message definition...
../data/../po/es.po:174: ...this is the location of the first definition
../data/../po/es.po:678: duplicate message definition...
../data/../po/es.po:178: ...this is the location of the first definition
../data/../po/es.po:683: duplicate message definition...
../data/../po/es.po:182: ...this is the location of the first definition
../data/../po/es.po:687: duplicate message definition...
../data/../po/es.po:186: ...this is the location of the first definition
../data/../po/es.po:691: duplicate message definition...
../data/../po/es.po:190: ...this is the location of the first definition
../data/../po/es.po:695: duplicate message definition...
../data/../po/es.po:194: ...this is the location of the first definition
../data/../po/es.po:699: duplicate message definition...
../data/../po/es.po:198: ...this is the location of the first definition
../data/../po/es.po:703: duplicate message definition...
../data/../po/es.po:202: ...this is the location of the first definition
../data/../po/es.po:707: duplicate message definition...
../data/../po/es.po:206: ...this is the location of the first definition
../data/../po/es.po:711: duplicate message definition...
../data/../po/es.po:212: ...this is the location of the first definition
../data/../po/es.po:715: duplicate message definition...
../data/../po/es.po:218: ...this is the location of the first definition
../data/../po/es.po:719: duplicate message definition...
../data/../po/es.po:223: ...this is the location of the first definition
../data/../po/es.po:723: duplicate message definition...
../data/../po/es.po:228: ...this is the location of the first definition
../data/../po/es.po:728: duplicate message definition...
../data/../po/es.po:232: ...this is the location of the first definition
../data/../po/es.po:733: duplicate message definition...
../data/../po/es.po:236: ...this is the location of the first definition
../data/../po/es.po:737: duplicate message definition...
../data/../po/es.po:240: ...this is the location of the first definition
../data/../po/es.po:741: duplicate message definition...
../data/../po/es.po:244: ...this is the location of the first definition
../data/../po/es.po:745: duplicate message definition...
../data/../po/es.po:248: ...this is the location of the first definition
../data/../po/es.po:749: duplicate message definition...
../data/../po/es.po:252: ...this is the location of the first definition
../data/../po/es.po:753: duplicate message definition...
../data/../po/es.po:256: ...this is the location of the first definition
../data/../po/es.po:757: duplicate message definition...
../data/../po/es.po:260: ...this is the location of the first definition
../data/../po/es.po:761: duplicate message definition...
../data/../po/es.po:264: ...this is the location of the first definition
../data/../po/es.po:765: duplicate message definition...
../data/../po/es.po:268: ...this is the location of the first definition
../data/../po/es.po:769: duplicate message definition...
../data/../po/es.po:272: ...this is the location of the first definition
../data/../po/es.po:773: duplicate message definition...
../data/../po/es.po:276: ...this is the location of the first definition
../data/../po/es.po:778: duplicate message definition...
../data/../po/es.po:280: ...this is the location of the first definition
../data/../po/es.po:782: duplicate message definition...
../data/../po/es.po:284: ...this is the location of the first definition
../data/../po/es.po:786: duplicate message definition...
../data/../po/es.po:288: ...this is the location of the first definition
../data/../po/es.po:790: duplicate message definition...
../data/../po/es.po:292: ...this is the location of the first definition
../data/../po/es.po:794: duplicate message definition...
../data/../po/es.po:296: ...this is the location of the first definition
../data/../po/es.po:798: duplicate message definition...
../data/../po/es.po:300: ...this is the location of the first definition
../data/../po/es.po:802: duplicate message definition...
../data/../po/es.po:304: ...this is the location of the first definition
../data/../po/es.po:806: duplicate message definition...
../data/../po/es.po:308: ...this is the location of the first definition
../data/../po/es.po:810: duplicate message definition...
../data/../po/es.po:312: ...this is the location of the first definition
../data/../po/es.po:814: duplicate message definition...
../data/../po/es.po:316: ...this is the location of the first definition
../data/../po/es.po:819: duplicate message definition...
../data/../po/es.po:320: ...this is the location of the first definition
../data/../po/es.po:824: duplicate message definition...
../data/../po/es.po:324: ...this is the location of the first definition
../data/../po/es.po:828: duplicate message definition...
../data/../po/es.po:328: ...this is the location of the first definition
../data/../po/es.po:832: duplicate message definition...
../data/../po/es.po:332: ...this is the location of the first definition
../data/../po/es.po:836: duplicate message definition...
../data/../po/es.po:336: ...this is the location of the first definition
../data/../po/es.po:840: duplicate message definition...
../data/../po/es.po:340: ...this is the location of the first definition
../data/../po/es.po:845: duplicate message definition...
../data/../po/es.po:344: ...this is the location of the first definition
../data/../po/es.po:850: duplicate message definition...
../data/../po/es.po:348: ...this is the location of the first definition
../data/../po/es.po:854: duplicate message definition...
../data/../po/es.po:352: ...this is the location of the first definition
../data/../po/es.po:858: duplicate message definition...
../data/../po/es.po:356: ...this is the location of the first definition
../data/../po/es.po:862: duplicate message definition...
../data/../po/es.po:360: ...this is the location of the first definition
../data/../po/es.po:866: duplicate message definition...
../data/../po/es.po:364: ...this is the location of the first definition
../data/../po/es.po:871: duplicate message definition...
../data/../po/es.po:368: ...this is the location of the first definition
../data/../po/es.po:875: duplicate message definition...
../data/../po/es.po:372: ...this is the location of the first definition
../data/../po/es.po:879: duplicate message definition...
../data/../po/es.po:376: ...this is the location of the first definition
../data/../po/es.po:883: duplicate message definition...
../data/../po/es.po:380: ...this is the location of the first definition
../data/../po/es.po:887: duplicate message definition...
../data/../po/es.po:384: ...this is the location of the first definition
../data/../po/es.po:892: duplicate message definition...
../data/../po/es.po:389: ...this is the location of the first definition
../data/../po/es.po:896: duplicate message definition...
../data/../po/es.po:393: ...this is the location of the first definition
../data/../po/es.po:901: duplicate message definition...
../data/../po/es.po:397: ...this is the location of the first definition
../data/../po/es.po:905: duplicate message definition...
../data/../po/es.po:401: ...this is the location of the first definition
../data/../po/es.po:910: duplicate message definition...
../data/../po/es.po:406: ...this is the location of the first definition
../data/../po/es.po:914: duplicate message definition...
../data/../po/es.po:410: ...this is the location of the first definition
../data/../po/es.po:918: duplicate message definition...
../data/../po/es.po:414: ...this is the location of the first definition
../data/../po/es.po:922: duplicate message definition...
../data/../po/es.po:418: ...this is the location of the first definition
../data/../po/es.po:926: duplicate message definition...
../data/../po/es.po:422: ...this is the location of the first definition
../data/../po/es.po:930: duplicate message definition...
../data/../po/es.po:426: ...this is the location of the first definition
../data/../po/es.po:934: duplicate message definition...
../data/../po/es.po:430: ...this is the location of the first definition
../data/../po/es.po:938: duplicate message definition...
../data/../po/es.po:434: ...this is the location of the first definition
../data/../po/es.po:942: duplicate message definition...
../data/../po/es.po:438: ...this is the location of the first definition
../data/../po/es.po:946: duplicate message definition...
../data/../po/es.po:442: ...this is the location of the first definition
../data/../po/es.po:951: duplicate message definition...
../data/../po/es.po:448: ...this is the location of the first definition
../data/../po/es.po:958: duplicate message definition...
../data/../po/es.po:455: ...this is the location of the first definition
../data/../po/es.po:965: duplicate message definition...
../data/../po/es.po:462: ...this is the location of the first definition
../data/../po/es.po:971: duplicate message definition...
../data/../po/es.po:467: ...this is the location of the first definition
../data/../po/es.po:975: duplicate message definition...
../data/../po/es.po:471: ...this is the location of the first definition
../data/../po/es.po:980: duplicate message definition...
../data/../po/es.po:476: ...this is the location of the first definition
../data/../po/es.po:985: duplicate message definition...
../data/../po/es.po:481: ...this is the location of the first definition
../data/../po/es.po:989: duplicate message definition...
../data/../po/es.po:485: ...this is the location of the first definition
../data/../po/es.po:993: duplicate message definition...
../data/../po/es.po:489: ...this is the location of the first definition
../data/../po/es.po:998: duplicate message definition...
../data/../po/es.po:495: ...this is the location of the first definition
../data/../po/es.po:1005: duplicate message definition...
../data/../po/es.po:502: ...this is the location of the first definition
../data/../po/es.po:1012: duplicate message definition...
../data/../po/es.po:507: ...this is the location of the first definition
../data/../po/es.po:1016: duplicate message definition...
../data/../po/es.po:511: ...this is the location of the first definition
../data/../po/es.po:1021: duplicate message definition...
../data/../po/es.po:516: ...this is the location of the first definition
../data/../po/es.po:1025: duplicate message definition...
../data/../po/es.po:520: ...this is the location of the first definition
../data/../po/es.po:1029: duplicate message definition...
../data/../po/es.po:524: ...this is the location of the first definition
../data/../po/es.po:1033: duplicate message definition...
../data/../po/es.po:528: ...this is the location of the first definition
../data/../po/es.po:1038: duplicate message definition...
../data/../po/es.po:533: ...this is the location of the first definition
../data/../po/es.po:1042: duplicate message definition...
../data/../po/es.po:537: ...this is the location of the first definition
msgfmt: found 122 fatal errors

```
